motoko/hello-angular-motoko/karma.conf.js

```diff
@@ -0,0 +1,44 @@
+// Karma configuration file, see link for more information
+// https://karma-runner.github.io/1.0/config/configuration-file.html
```

Comment:
This is a testing framework, right? Can we add 1-2 tests that the project still works as part of CI? This would make it much easier to maintain in the future (e.g. updating dependencies).

Reply:
Added a Test Suite for the motoko service.

Reply:
@sesi200, Yes. This file just needs to be at the project root with 
1) `motoko-classes-example-...` replaced with this examples corresponding name?
2) The `run:` commands replaced with the appropriate commands?

Reply:
yep, that's it

Reply:
exactly, that's it